src/Symfony/Bundle/WebProfilerBundle/Controller/ExceptionController.php

```diff
@@ -30,14 +30,18 @@ class ExceptionController
     protected $twig;
     protected $debug;
     protected $profiler;
+    private $charset;
     private $fileLinkFormat;
+    private $errorRenderer;
 
-    public function __construct(Profiler $profiler = null, Environment $twig, bool $debug, FileLinkFormatter $fileLinkFormat = null)
+    public function __construct(Profiler $profiler = null, Environment $twig, bool $debug, FileLinkFormatter $fileLinkFormat = null, string $charset = null, HtmlErrorRenderer $errorRenderer = null)
     {
         $this->profiler = $profiler;
         $this->twig = $twig;
         $this->debug = $debug;
+        $this->charset = $charset;
```

Comment:
This should be below the next variable to follow the signature order or is there a reason behind it? Same for the order of the  variables